README.md

```diff
@@ -75,6 +75,37 @@ We recommend developers use Seq locally to help ensure the audit event developme
 
 ![Sample Seq Output](./assets/seq-events-view.png)
 
+## Using custom sink for large audit events
+
+With increasing size in audit events large audit events couldn't be processed by `Kmd.Logic.Audit.Client.SerilogAzureEventHubs.SerilogAzureEventHubsAuditClient` implementation. As a solution to this we came up with a custom sink which based on the size of audit event will push the event to blob for large messages and keep the blob url in event hub. The default bytes limit is 256 KB which is customizable.
```

Comment:
You have to mention `SerilogLargeAuditEvents` in line 22 of this doco too. As we will two implementation of `IAudit` that use Logic backend